bedrock/firefox/tests/test_base.py

```diff
@@ -375,6 +375,19 @@ def test_fx_82_0_0_china(self, render_mock):
 
     # end 82.0 whatsnew tests
 
+    # begin 83.0 whatsnew tests
+
```

Comment:
We're going to localize this WNP for a specific set of languages (that match the same l10n from Pocket). Should we test the routing works for those specific languages?

Reply:
How long is the list? If it's 3, test them all. If it's 20 I think it's enough to test EN and one other local expected to display 83 and one other expected not too.

Reply:
Oh, the list of which are supported and not is managed by l10n. I don't think you can write a test for this.... hold please.

Reply:
Okay, I think what you want to do is write a test for `en-US`, and then one for a different local, like `de` and test it with `ftl_file_is_active` set to `True` and to `False` to check the fallback works as expected.

Reply:
Or you can copy the test for 81 or 80, since I think those pages show up under the same conditions.

Reply:
@pmac Am I right about this? Can we specify if the language file is active [in a test like this](https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/tests/test_base.py#L336).

Reply:
Do you mean you just want to test that our localization system works for this page? If so I don't think we need to do that. We have tests that prove that pages can be activated in languages and that l10n works. If there is locale specific template choosing in the view then that can be tested of course. Is that what we're saying? From the view code it looks like all we care about is that the Fluent file is active, so I don't think a test is needed beyond what @stephaniehobson suggested with th emock of `ftl_file_is_active` and making sure it does what you expect when that returns `True` and `False`.